lua/lean/_util.lua

```diff
@@ -50,4 +50,33 @@ function M.subprocess_check_output(opts, timeout)
   ))
 end
 
+function M.wrap_handler(other_handler, handler)
+  return function(...)
+    other_handler(...)
+    handler(...)
+  end
+end
+
+function M.mk_handler(fn)
```

Comment:
This looks like it's @mfussenegger's code from https://github.com/mfussenegger/nvim-lsp-compl/commit/29a81f3747373f42d002775b1f79dcd19fada2e4 right? If so it's GPL 3.0 so we at very least have to notate that here (unless perchance @mfussenegger has already said somewhere it's OK to crimp regardless?)

Reply:
Yep just copied it -- added acknowledgements, let me know it that is sufficient

Reply:
Fine for me to copy it.

Reply:
Thank you (both)!